lib/rubygems/core_ext/kernel_warn.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+if RUBY_VERSION >= "2.5"
+  module Kernel
+    path = "#{__dir__}/"
+    original_warn = instance_method(:warn)
+    Module.new {define_method(:warn, original_warn)}
+    original_warn = method(:warn)
+
+    module_function define_method(:warn) {|*messages, uplevel: nil|
+      if uplevel
+        uplevel, = [uplevel].pack("l!").unpack("l!")
```

Comment:
What is this even doing?

Reply:
Converting `uplevel` to an `Integer` which fits `long`, with calling `to_int` if needed.
There is no such method right now.